modules/ROOT/pages/rtf-permissions.adoc

```diff
@@ -0,0 +1,27 @@
+== Runtime Fabric Core Software Permissions
+
+The following table lists all the Kubernetes resource access authorizations used by Runtime Fabric core software.
+These RBAC (Role-based access control) configurations (ClusterRole and Rolebindings) are part of the Runtime Fabric helm charts and are automatically applied during the installation.
+
+[NOTE]
+You can't extend or modify any of these core software access permissions. You need to create and apply the ClusterRole and Rolebindings for the Runtime Fabric Authorized namespaces use case. Refer to xref:authorized-namespaces.adoc[].
```

Comment:
Not sure. if we care about "You need to". Also not sure about formatting of "ClusterRole and Rolebindings".

```suggestion
You can't extend or modify any of these core software access permissions. Instead, create and apply the ClusterRole and Rolebindings for the Runtime Fabric Authorized namespaces use case. Refer to xref:authorized-namespaces.adoc[].
```